pkg/crc/cluster/cert-renewal.go

```diff
@@ -1,73 +1,72 @@
 package cluster
 
 import (
+	"errors"
 	"fmt"
-	"regexp"
+	"strings"
 	"time"
 
-	"github.com/code-ready/crc/pkg/crc/errors"
+	crcerrors "github.com/code-ready/crc/pkg/crc/errors"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/oc"
 	"github.com/code-ready/crc/pkg/crc/ssh"
-	"github.com/code-ready/crc/pkg/crc/systemd"
 )
 
-func waitForPendingCsrs(ocConfig oc.Config) error {
-	waitForPendingCsr := func() error {
+func waitForPendingCsrs(ocConfig oc.Config, signerName string) error {
```

Comment:
Since you are changing the method, you can fix the spelling `waitForPendingCSRs` to be consistent with the other method with `CSRs` in its name.

Reply:
done